components/fleetArchiveDS.js

```diff
@@ -147,9 +147,6 @@ class FleetArchiveLogs extends Datasource {
       .uniq()
       .compact()
       .value();
-    if (vehicleIds.length === 0) {
```

Comment:
What does this fix? It's a bit unclear.

Reply:
You are right. This doesn't actually fix anything. It just removes an early exit.